### PR TITLE
create-dmg 1.0.0.7

### PR DESCRIPTION
**Pull request by aonez**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by fxcoudert** [comment]
Catalina test failure:

```
/var/folders/07/mmpkd6_x7fvbpl_msv10c7fh0000gp/T/createdmg.tmp.XXXXXXXXXX.YcU2D0uu:85:89: execution error: Finder got an error: A privilege violation occurred. (-10004)
```

**Comment by aonez** [comment]
Locally that execution works, not sure why it fails in the jenkins Catalina test.

File: Formula/create-dmg.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class CreateDmg < Formula
   desc "Shell script to build fancy DMGs"
   homepage "https://github.com/andreyvit/create-dmg"
-  url "https://github.com/andreyvit/create-dmg/archive/v1.0.0.5.tar.gz"
-  sha256 "de76c8a7a1f4705720d61d39de7c87b7bc2acc7c35f6ec8d6d2dbdafcedc21b6"
+  url "https://t.keka.io/create-dmg.1.0.0.6.zip"
```
Comment (fxcoudert) [inline Formula/create-dmg.rb]: No, please use upstream distribution.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/create-dmg.rb (hunk 1 above)
Comment (aonez) [reply Formula/create-dmg.rb]: For sure! Just testing how to solve the previous Catalina test error without releasing it in upstream.

**Comment by fxcoudert** [comment]
We need to understand the cause of the failure, rather than just skip some code during testing. Is `osascript` failing? Why? Is that something we should fix in brew?

**Comment by aonez** [comment]
@fxcoudert `osascript` is falling indeed, only on Catalina test. I've tried to figure out with no luck, not having access to that test machine makes it difficult for me. Locally it works.

I needed to push this update because 1.0.0.5 fails on Catalina, that's why ultimately made the skip.

**Comment by aonez** [comment]
Searching for the error found that maybe the test machine is not running the `osascript` on Finder and therefore it fails. But can't check that.

**Comment by fxcoudert** [comment]
I see:

```
/usr/local/Cellar/create-dmg/dmg/bin/create-dmg: line 252: [: -gt: unary operator expected
```

(which may be related to the change in default shell in Catalina?) and:

```
execution error: Finder got an error: A privilege violation occurred. (-10004)
```

which may be a sandbox issue.

**Comment by aonez** [comment]
The -gt was an innocuous bug that I’ve fixed in the latest version. Was checking an empty variable.

**Comment by aonez** [comment]
Maybe can I check the sandbox profile the test machine is using?

`/usr/bin/sandbox-exec -f /private/tmp/homebrew20191015-57549-1slx58z.sb`

**Comment by mRs-** [comment]
Is there any Kind of Progress for this?

**Pull request closed without merging**